composer.json

```diff
@@ -13,20 +13,20 @@
     "require": {
         "php": "^7.1.3",
         "composer/composer": "^1.0",
-        "illuminate/console": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/container": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/contracts": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/database": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/http": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/pipeline": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/support": "5.6.*|5.7.*|5.8.*|^6.0",
+        "illuminate/console": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
+        "illuminate/container": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
+        "illuminate/contracts": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
+        "illuminate/database": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
+        "illuminate/http": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
+        "illuminate/pipeline": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
+        "illuminate/support": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
         "mockery/mockery": "^1.0|0.9.*",
         "phpstan/phpstan": "^0.11.15",
-        "symfony/process": "^4.2",
+        "symfony/process": "^4.2|^5.0",
         "ext-json": "*"
     },
     "require-dev": {
-        "orchestra/testbench": "^3.8",
+        "orchestra/testbench": "^3.6|dev-master",
```

Comment:
No version of testbench will work properly yet, until https://github.com/orchestral/testbench-core/pull/32 is merged.